doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1162 @@
+           NVO Name  : nvo1,  VTEP : VTEP1
+           Source interface  : Loopback33
+
+2. show vxlan vlanvnimap 
+   - Displays all the VLAN VNI mappings.
+
+   +---------+-------+
+   | VLAN    |   VNI |
+   +=========+=======+
+   | Vlan100 |   100 |
+   +---------+-------+
+   | Vlan101 |   101 |
+   +---------+-------+
+   Total count : 2
+
+
+3. show vxlan vrfvnimap 
+   - Displays all the VRF VNI mappings.
+
+4. show vxlan tunnel
+   - lists all the discovered tunnels.  
+   - SIP, DIP, Creation Source, OperStatus are the columns.
+
+   +---------+---------+-------------------+--------------+
+   | SIP     | DIP     | Creation Source   | OperStatus   |
+   +=========+=========+===================+==============+
+   | 2.2.2.2 | 4.4.4.4 | EVPN              | oper_up      |
+   +---------+---------+-------------------+--------------+
+   | 2.2.2.2 | 3.3.3.3 | EVPN              | oper_up      |
+   +---------+---------+-------------------+--------------+
+   Total count : 2
+
+5. show vxlan remote_mac <remoteip/all>  <vlanid/all> 
+   - lists all the MACs learnt from the specified remote ip or all the remotes for the specified/all vlans. (APP DB view) 
+   - VLAN, MAC, RemoteVTEP,  VNI,  Type are the columns.
+
+   show vxlan remote_mac all
+   +---------+-------------------+--------------+-------+--------+
+   | VLAN    | MAC               | RemoteVTEP   |   VNI | Type   |
+   +=========+===================+==============+=======+========+
+   | Vlan101 | 00:00:00:00:00:01 | 4.4.4.4      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:02 | 3.3.3.3      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:03 | 4.4.4.4      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:04 | 4.4.4.4      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:05 | 4.4.4.4      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:99 | 3.3.3.3      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   Total count : 6
+   
+   show vxlan remote_mac 3.3.3.3
+   +---------+-------------------+--------------+-------+--------+
+   | VLAN    | MAC               | RemoteVTEP   |   VNI | Type   |
+   +=========+===================+==============+=======+========+
+   | Vlan101 | 00:00:00:00:00:02 | 3.3.3.3      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   | Vlan101 | 00:00:00:00:00:99 | 3.3.3.3      |  1001 | static |
+   +---------+-------------------+--------------+-------+--------+
+   Total count : 2
+
+
+6. show vxlan remote_vni <remoteip/all> 
+   - lists all the VLANs learnt from the specified remote ip or all the remotes.  (APP DB view) 
```

Comment:
This needs to be rephrased to say "lists all the VNIs learnt from" ?? VLAN is local and mapped to a VNI